### DISCUSSION TITLE
Bootstrap icon in description?

### DISCUSSION THREAD

**Discussion by aidanastridge** [quarto-dev/quarto-cli#10357] [Q&A] [answered]
### Description

```python
---
title: "Title"
description: "{Bootstrap icon} Location <br/ > [GitHub]() [LinkedIn]() [X]()"
---
``` 
Hey Quarto,

I've tried placing a bootstrap icon in a quarto description using html, however it doesn't publish/run. Quarto recognizes a break tag in the description.

**Comment by mcanouil** [comment, marked as answer]
You need to use a shortcode extension for icon, see the Quarto Iconify extension for example.

> Quarto recognizes a break tag in the description.

You put it there.

**Reply by aidanastridge** [reply]
Thanks! Maybe I wrote my question wrong, but I knew I placed a break tag in there; I  assumed all html syntax would work.

**Reply by mcanouil** [reply]
You should not have bare raw contents because it means you want the tools to guess what it is in context where the tools expect markdown.
YAML allows multi lines.
```yml
key: |
  Whatever followed by two spaces to have a line break  
  A second line here.
```

If you really want raw content, tell Quarto what it is, see <https://quarto.org/docs/authoring/markdown-basics.html#raw-content>.
